.github/workflows/functional-test.yaml

```diff
@@ -533,10 +566,19 @@ jobs:
         with:
           name: ${{ matrix.name }}_container_logs
           path: ./${{ env.RADIUS_CONTAINER_LOG_BASE }}
+      - name: Delete azure resource group - ${{ env.AZURE_TEST_RESOURCE_GROUP }}
+        if: always()
+        run: |
+          # if deletion fails, purge workflow will purge the resource group and its resources later.
+          az group delete \
```

Comment:
Do we need force flag (if there is any) so that we don't end up with dangling resource groups and resources?

Reply:
What is a dangling resource group or resource?

Reply:
Was some rebase issue with main